improver_tests/acceptance/acceptance.py

```diff
@@ -84,6 +89,125 @@ def cli_name_with_dashes(dunder_file):
     return module_dashes
 
 
+@functools.lru_cache()
+def acceptance_checksums(checksum_path=None):
+    """
+    Retrieve a list of checksums from file in text list format, as produced by
+    the sha256sum command line tool.
+
+    Args:
+        checksum_path (Optional[pathlib.Path]): Path to checksum file. File
+            should be plain text in the format produced by the sha256sum
+            command line tool. Paths listed in the file should be relative to
+            the KGO root directory found by kgo_root().
+
+    Returns:
+        Dict[pathlib.Path, str]: Dict with keys being relative paths and
+            values being hexadecimal checksums
+    """
+    if checksum_path is None:
+        checksum_path = DEFAULT_CHECKSUM_FILE
+    if not kgo_exists():
+        raise IOError("KGO path must be defined")
+    with open(checksum_path, mode="r") as checksum_file:
+        checksum_lines = checksum_file.readlines()
+    checksums = {}
+    for line in checksum_lines:
+        parts = line.strip().split("  ", maxsplit=1)
+        csum = parts[0]
+        path = pathlib.Path(parts[1])
+        checksums[path] = csum
+    return checksums
+
+
+def verify_checksum(kgo_path, checksums=None, checksum_path=None):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Useless return at end of function or method](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5572303)